### DISCUSSION TITLE
Copilot in IntelliJ IDEA encountered a error when logging in to server

### DISCUSSION THREAD

**Discussion by lyjwolf318** [community/community#17282] [Copilot Conversations]
2022/5/26
1:11	GitHub Copilot: Failed to initiate the GitHub login process. Please try again.

**Comment by lyjwolf318** [comment]
the problem is solved

**Reply by Raxcl** [reply]
How did it work out?

**Reply by litt1e-p** [reply]
+1

**Reply by 24sama** [reply]
+1

**Reply by kyokutor** [reply]
+1

**Reply by litt1e-p** [reply]
I have solved it by change dns to 8.8.8.8

**Reply by kyokutor** [reply]
> 

useless

**Reply by fengyikang88** [reply]
> 问题已经解决了

怎么解决的呀

**Comment by lyjwolf318** [comment]
use dnsjumper app to flush dns and set dns 8.8.8.8, then it works.

**Reply by midFang** [reply]
I set it this way, but that's fine
